go-client/README.md

```diff
@@ -0,0 +1,49 @@
+# Deephaven Go Client
+
+This is a Go package that provides a client interface for [Deephaven Community Core](https://github.com/deephaven/deephaven-core).
+Details on usage can be found in the examples and in the documentation (both listed below).
+
+## Setup
+
+First, you will need a working Go install (1.13+ is required; so apt-get is often too outdated).
+It can be installed from [the official site](https://go.dev/doc/install).
+
+In order to run tests or examples, a Deephaven server must be running. 
+To run a server, follow the instructions [here](https://github.com/deephaven/deephaven-core#run-deephaven).
+
+## Tests
+
+To run tests, simply run the following command from this folder.
+```
+$ go test -v ./...
+```
+All tests should pass within 10 seconds.
+Tests assume the server is on localhost:10000 by default, but can be configured by setting the DH_HOST and DH_PORT environment variables.
+
+## Examples
+
+This project also includes a several example binaries.
+Examples assume the server is running on localhost:10000.
+An example can be run using the following command:
+```
+$ go run ./examples/import_table
+```
+Other valid paths for examples are
+- `./examples/query`
+- `./examples/fetch_table`
+- `./examples/import_table`
+- `./examples/input_table`
+- `./examples/run_script`
```

Comment:
```suggestion
An example can be run using one of the following command:
```bash
$ go run ./examples/import_table
$ go run ./examples/query
$ go run ./examples/fetch_table
$ go run ./examples/import_table
$ go run ./examples/input_table
$ go run ./examples/run_script
```

Reply:
make sure query gets renamed to basic_query, if that happens.